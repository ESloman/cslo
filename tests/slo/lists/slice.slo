var x = [10, 20, 30, 40, 50];
println(x[1:4]); // Should print [20, 30, 40]
println(x[:3]);  // Should print [10, 20, 30]
println(x[2:]);  // Should print [30, 40, 50]
println(x[-3:-1]); // Should print [30, 40]
