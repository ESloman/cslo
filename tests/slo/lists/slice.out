list[3]: [20, 30, 40]
list[3]: [10, 20, 30]
list[3]: [30, 40, 50]
list[2]: [30, 40]
